src/controllers/PartialUserFormController.php

```diff
@@ -51,20 +51,18 @@ class PartialUserFormController extends UserDefinedFormController
      */
     public function partial(HTTPRequest $request)
     {
-        // Ensure this URL doesn't get picked up by HTTP caches
-        HTTPCacheControlMiddleware::singleton()->disableCache();
-
         /** @var PartialFormSubmission $partial */
         $partial = $this->validateToken($request);
-        if ($this->data()->PasswordProtected &&
-            $request->getSession()->get(PasswordForm::PASSWORD_SESSION_KEY) !== $partial->ID
-        ) {
-            return $this->redirect('verify');
-        }
         $record = DataObject::get_by_id($partial->UserDefinedFormClass, $partial->UserDefinedFormID);
         /** @var self $controller */
         $controller = self::create($record);
         $controller->doInit();
+        $controller->dataRecord = $record;
```

Comment:
But the constructor already set the dataRecord 🤔

Reply:
Yep, but it doesn't work without this.... Don't ask me why

Reply:
Doesn't work in what way?